src/morphometrics/segment/post_process.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional
+
+import numpy as np
+from skimage import measure
+from skimage.transform import resize
+
+
+def post_process_image(
+    segmented_image: np.ndarray,
+    threshold: int,
+    segmentation_mask: Optional[np.ndarray] = np.zeros(1),
```

Comment:
```suggestion
    segmentation_mask: Optional[np.ndarray] = None,
```

It is more standard to make the default parameter `None` when the argument is optional. Please also update the relevant `if` statement below.